.pipelines/templates/aks-engine-setup.yaml

```diff
@@ -0,0 +1,70 @@
+steps:
+  - script: |
+      curl -LO https://github.com/Azure/aks-engine/releases/download/v${AKS_ENGINE_VERSION}/aks-engine-v${AKS_ENGINE_VERSION}-linux-amd64.tar.gz
+      tar -zxvf aks-engine-v${AKS_ENGINE_VERSION}-linux-amd64.tar.gz
```

Comment:
should we use https://aka.ms/aks-engine/aks-engine-k8s-e2e.tar.gz

Reply:
Does it always host the latest version of aks-engine? Currently I've configured the pipeline to use `v0.49.0.`

Reply:
Right now it's pointing to `v0.48.0` but let's confirm with @chewong to see how often that gets updated to the latest.

Reply:
We can merge as is and replace it later.